src/app/components/feed/FeedImportDialog.js

```diff
@@ -114,7 +114,7 @@ const FeedImportDialog = ({
         open
       >
         <div className={dialogStyles['dialog-title']}>
-          <div>
+          <div className={dialogStyles['dialog-title-choice']}>
```

Comment:
unrelated to the functionality in this PR, but as I was grabbing the styles to use the toggle buttons in the new merge dialog, figure I would just add this class so the styles/layout matches for both use cases